<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>671</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bangla MN</family>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt;

color: rgb(0,0,0);</string>
    </property>
    <property name="text">
     <string>ELENCO DELLE OPERAZIONI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancella">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
font: 14pt;
border: 1px solid rgb(176, 141, 48);</string>
    </property>
    <property name="text">
     <string>PULISCI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="carica">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>241</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(176, 141, 48);
font: 14pt;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>CARICA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEsci">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>551</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
font: 14pt;
border: 1px solid rgb(176, 141, 48);</string>
    </property>
    <property name="text">
     <string>ESCI</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>210</y>
      <width>805</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255,255,255);
</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>311</width>
      <height>461</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
    <property name="batchSize">
     <number>100</number>
    </property>
    <item>
     <property name="text">
      <string>ore di lavoro dei dipendenti</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>entrate annuali</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>canoni di manutenzione</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>numero defunti negli anni</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>et√† media dei defunti</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>numero di allacci</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>numero di distacchi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>numero totale di allacci e di distacchi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>defunti senza incaricato</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>numero di manutenzioni</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>entrate annuali dai soli servizi aggiuntivi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>clienti che devono pagare una fattura</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>defunti che sono stati riallacciati</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>entrate totali</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../source.qrc"/>
 </resources>
 <connections/>
</ui>
